stetho/src/main/java/com/facebook/stetho/inspector/helper/ObjectIdMapper.java

```diff
@@ -0,0 +1,79 @@
+// Copyright 2004-present Facebook. All Rights Reserved.
+
+package com.facebook.stetho.inspector.helper;
+
+import android.util.SparseArray;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.annotation.concurrent.GuardedBy;
+
+public class ObjectIdMapper {
```

Comment:
Do we want it to be generic? I didn't need it to be generic right now, so seemed kind of pointless for now, and trivial cost to change later ...